patch/driver-support-optoe-oneaddr-pageable.patch

```diff
@@ -0,0 +1,64 @@
+From cb4523bb02cb6228e3e8e5f3333e6af9a1b15466 Mon Sep 17 00:00:00 2001
```

Comment:
What is this patch for? If this is a newly added patch shall we create a separated PR?

Reply:
sorry, i've rebased my master to latest. Can you review now?